CHANGELOG.md

```diff
@@ -2,8 +2,10 @@
 
 ### To be Released
 
-* fix(autocomplete): do not show update message on autocomplete
-* feat(databases): add databases users update
+### 1.30.2
+
+* fix(autocomplete): do not show update message on autocomplete ([PR#1044](https://github.com/Scalingo/cli/pull/1044))
+* feat(databases): add databases users update ([PR#1036](https://github.com/Scalingo/cli/pull/1036))
```

Comment:
suggestion: with a new feature in this release, I would strictly follow semver and release version 1.31.0. What do you think?

Reply:
For me it completed the user management implemented with 1.30.0:
```
### 1.30.0

* feat(addons): add user management commands ([PR#1019](https://github.com/Scalingo/cli/pull/1019))
```
But you're right it's an additional feature.

I'm closing this PR to recreate the correct branch

Reply:
Here it is https://github.com/Scalingo/cli/pull/1050